rules.d/50-udev-default.rules.in

```diff
@@ -14,6 +14,8 @@ SUBSYSTEM=="virtio-ports", KERNEL=="vport*", ATTR{name}=="?*", SYMLINK+="virtio-
 SUBSYSTEM=="rtc", ATTR{hctosys}=="1", SYMLINK+="rtc"
 SUBSYSTEM=="rtc", KERNEL=="rtc0", SYMLINK+="rtc", OPTIONS+="link_priority=-100"
 
+SUBSYSTEM=="hidraw", IMPORT{builtin}="usb_id", IMPORT{builtin}="hwdb"
```

Comment:
Why "usb_id" is called? At least, please do not import usb_id unconditionally, but conditionalize with `SUBSYSTEMS=="usb", if it really necessary.